### ISSUE TITLE
Eight Azure Kinects on One Workstation

### ISSUE THREAD

**Issue by DanTebbsFRL** [microsoft/Azure-Kinect-Sensor-SDK#1756]
We have an application where we need to connect qty 8, Azure Kinect sensors to a single workstation computer which integrates all 8 device streams into a recording.

After much experimentation, we found that we can always connect 4 AK sensors for both:
HiPoint (Rocket U 1388C, 8-Port, powered), and
Startech (PEXUSB3S44V, 4-Port, powered) x2 PCI-E cards.

The first 4 will always come up correctly using various ports, drivers, driver settings, cables, and power sources. The 5th will usually come up correctly as well (~ 95% of the time).

However, then things quickly become more unreliable while trying to get up to 8 devices. When plugging in the 6th device, the enumeration of all 6 AK devices only works about 75% of the time. Sometimes the 6th one appears, but one of 1~5 goes away. For the 7th device we get all 7 maybe 20% of the time. We only ever got all 8 devices maybe 5 times total in all the experimenting we did.

Once all 8 AKs enumerate correctly, we find that we can collect all 8 depth streams and all 8 image streams just fine.

As we worked with these various configurations, we began to suspect that the enumeration time itself was the problem. It was observed that because the AK has internal USB hubs, a color camera, a depth camera, and the microphone array, and that previously enumerated AKs re-enumerate... there is a huge amount of enumeration going on as you approach 8 AKs.

It seems likely that all this enumeration is happening to USB spec, but that enumerating all 6, 7 or 8 at once, leads to the driver or the OS concluding that one or more devices is non-responsive, and then canceling the enumeration. This theory was supported by turning off an optional feature (which no longer needed to be enumerated) at the driver level, and observing that chances of successful enumeration went up.

In summary: 
We request a fix or new feature to allow 8 AKs, and suggest 2 possible approaches. One may be to allow an increase in the time-out allotted during enumeration. Another fix may be to change the driver to not re-enumerate those devices which have already enumerated successfully.
